plugins/servicenow/icon_servicenow/actions/search_incident_attachment/action.py

```diff
@@ -29,9 +29,8 @@ def run(self, params={}):
 
         try:
             results = response["resource"].get("result")
-        except KeyError as e:
-            raise PluginException(preset=PluginException.Preset.UNKNOWN, data=response.text) from e
-
-        attachment_ids = [result.get("sys_id") for result in results]
+            attachment_ids = [result.get("sys_id") for result in results if isinstance(result, dict)]
+        except Exception as error:
```

Comment:
Have you tested which errors are most likely to appear? Is `Exception` to broad of a statement here?

Reply:
Before it was a `KeyError`, but `AttributeError` anyway. I think that's fine to catch any Exception, and raise `PluginException` instead